src/System.Console/tests/Performance/Perf.Console.cs

```diff
@@ -85,5 +84,77 @@ public void OpenStandardError()
                     s.Dispose();
             }
         }
+
+        [Benchmark]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
You've made these platform specific because of the call to get_ForegroundColor/get_BackgroundColor?  How about just using ConsoleColor.ResetColor() at the end instead?